etc/docker/elyra/Dockerfile

```diff
@@ -39,9 +39,11 @@ RUN python -m pip install --upgrade pip && \
     npm install -g yarn && \
     npm install -g npm && \
     cd /tmp && git clone https://github.com/elyra-ai/elyra.git && \
-    cd /tmp/elyra && git checkout tags/v1.0.0rc3 -b v1.0.0rc3 && make install
+    cd /tmp/elyra && git checkout tags/v1.0.0rc2 -b v1.0.0rc2 && make install
 
 WORKDIR /home/jovyan/work
 
-CMD ["/usr/local/bin/start-elyra.sh"]
+# set password to "password"
+ADD jupyter_notebook_config.json /home/jovyan/.jupyter/
```

Comment:
how would this help? does this work on your system because you have this key stored/saved on your browser? What happens when someone tries this for the first time and that it does not see the URL with the key to use to access the notebook UI?

Reply:
he just uses "password" as password, without the need to copy-paste the token. also in "deamon" mode, getting the token is even more complicated. that way it can be stored in the browser.

Reply:
please let me know if you disagree and we can go pack to the token based auth again

Reply:
now moved back to jupyterhub and set user to jovyan and password to jupyter which apears to be the default, is this ok?